src/components/Book.js

```diff
@@ -1,16 +1,16 @@
 import { useDispatch } from 'react-redux';
 import './Book.scss';
 import { PropTypes } from 'prop-types';
-import { removeBook } from '../redux/books/books';
+import { removeBook, fetchBooks } from '../redux/books/books';
 
 const Book = ({
   genre, author, title, id,
 }) => {
   const dispatch = useDispatch();
   const onClickRemove = () => {
+    console.log('hello', id);
```

Comment:
- [ ] we are discouraged not to commit `console.log` to our repo. Because It’s ugly, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website, kindly remove all the `console.log` in the project.

Reply:
Thank you for your observation!